<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifywindow</class>
 <widget class="QDialog" name="modifywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Information</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/Imagenes/BookSeller-Logo.png</normaloff>:/Imagenes/BookSeller-Logo.png</iconset>
  </property>
  <widget class="QLabel" name="fondoLabel">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-2</y>
     <width>321</width>
     <height>205</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/Imagenes/Hoja.JPG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLE">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>New User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLE">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>New Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mailLE">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>New Mail</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changePB">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
